doc/cephfs/cephfs-mirroring.rst

```diff
@@ -395,6 +395,37 @@ This allows a user to add a non-existent directory for synchronization. The mirr
 will mark such a directory as failed and retry (less frequently). When the directory is
 created, the mirror daemon will clear the failed state upon successful synchronization.
 
+Adding a new snapshot or a new directory manually in the .snap directory of the remote filesystem will result in failed status of the corresponding configured directory. In the remote filesystem::
+
+  $ ceph fs subvolume snapshot create cephfs subvol1 snap2 group1
+  or
+  $ mkdir /d0/.snap/snap2
+
+  $ ceph --admin-daemon /var/run/ceph/cephfs-mirror.asok fs mirror peer status cephfs@360 a2dc7784-e7a1-4723-b103-03ee8d8768f8
+  {
+    "/d0": {
+        "state": "failed",
+        "failure_reason": "snap_path=/d0/.snap/snap2 has invalid metadata in remote snapshot",
```

Comment:
`snap_path=/d0/.snap/snap2` is redundant since this information is under `/d0` section. Suggest changing this to `snapshot "snap2" has invalid metadata`

Reply:
@vshankar Fixed